cpu/saml1x/include/periph_cpu.h

```diff
@@ -50,6 +50,9 @@ enum {
 /** @} */
 
 #ifndef DOXYGEN
+/**
+ * @brief   ADC resolution values.
+ */
```

Comment:
Note that this is hidden from Doxygen anyway. (IMO this is sensible to do for all `adc_rec_t` definitions except for the default in `drivers/include/periph/adc.h`.)

Maybe it makes more sense to just drop the doc for this type instead?

Reply:
To be honest it's easier to maintain that in a single place file, in `periph/adc.h`.

Reply:
Removed the doc in https://github.com/RIOT-OS/RIOT/pull/16813/commits/8a7a51588e519a50f06ad825a8a6303fc0ca82f6